src/field_theory/finite/basic.lean

```diff
@@ -17,15 +17,16 @@ This file contains basic results about finite fields.
 Throughout most of this file, `K` denotes a finite field
 and `q` is notation for the cardinality of `K`.
 
+See `ring_theory.integral_domain` for the fact that the unit group of a finite field is a
```

Comment:
```suggestion
See `ring_theory/integral_domain` for the fact that the unit group of a finite field is a
```
It's unfortunate that we need to use `.` in the import statements. But here I think a `/` is better, it makes clear that you are referring to a file, and not a namespace.declaration